common/culture/innovations/wc_regional_innovations.txt

```diff
@@ -2,18 +2,32 @@
 @camel = "gfx/interface/icons/regional_innovations/innovation_war_camels.dds"
 @desert = "gfx/interface/icons/regional_innovations/innovation_desert_warfare.dds"
 @elekk = "gfx/interface/icons/regional_innovations/innovation_elephantry.dds"
+@dragonhawk = "gfx/interface/icons/regional_innovations/innovation_dragonhawk.dds"
 @gryphon = "gfx/interface/icons/regional_innovations/innovation_gryphon.dds"
 @guerilla = "gfx/interface/icons/regional_innovations/innovation_guerilla_warfare.dds"
 @kalimdor = "gfx/interface/icons/regional_innovations/innovation_kalimdor_settling.dds"
 @kodo = "gfx/interface/icons/regional_innovations/innovation_kodo.dds"
 @kyparite_sap = "gfx/interface/icons/regional_innovations/innovation_kypartie_sap.dds"
+@mushan = "gfx/interface/icons/regional_innovations/innovation_mushan.dds"
 @sea_faring = "gfx/interface/icons/regional_innovations/innovation_sea_faring.dds"
 @taiga = "gfx/interface/icons/regional_innovations/innovation_taiga_warfare.dds"
 @wyvern = "gfx/interface/icons/regional_innovations/innovation_wyvern.dds"
 
 
 ###Men-at-arms
 ##Tribal
+innovation_arachnathid = {
+	group = culture_group_regional
+	culture_era = culture_era_high_medieval
+	region = world_innovation_arachnathid
+	icon = @taiga
+	
+	unlock_maa = arachnathid
```

Comment:
Need tech and maa illustrations.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/151370066-abbd92dc-845d-4495-a60f-da956dac6ce7.png)
![image](https://user-images.githubusercontent.com/46384992/151370086-bf4a3353-4a31-4e98-92a8-bea5c80aa470.png)

</details>